lib/plugins/package/index.js

```diff
@@ -22,7 +22,7 @@ class Package {
         .then(this.validate)
         .then(this.zipService),
 
-      'after:deploy:deploy': () => BbPromise.bind(this)
+      'deploy:clean': () => BbPromise.bind(this)
```

Comment:
Could you move this above `deploy:createDeploymentArtifacts` so that correct order the hooks run through can be seen without looking into the `deploy` plugin.